src/analytics/segment.js

```diff
@@ -52,8 +53,7 @@ const extraProperties = store => {
     androidVersionCode: getAndroidVersionCode(VersionNumber.buildVersion),
     androidArchitecture: getAndroidArchitecture(VersionNumber.buildVersion),
     environment: ANALYTICS_LOGS ? "development" : "production",
-    localeIdentifier,
-    preferredLanguage: preferredLanguages ? preferredLanguages[0] : null,
+    systemLanguage,
```

Comment:
Adapting to the change of library:
- **`src/analytics/segment.js`**:
     In the analytics global properties:
    - removing property `localeIdentifier` which was irrelevant on iOS
    - removing property `preferredLanguage` which was working only on iOS
    - adding a property `systemLanguage` which has better naming, works on iOS